climakitae/data_export.py

```diff
@@ -0,0 +1,225 @@
+import xarray as xr
+import pandas as pd
+import shutil
+import dask
+import warnings
+import datetime
+import numpy as np
+xr.set_options(keep_attrs=True)
+import os
+
+def _export_to_user(user_export_format,data_to_export,
+                    file_name,**kwargs):
+    """
+    The data export method, called by core.Application.export_dataset. Saves
+    a dataset to the current working directory in the output format requested by the user 
+    (which is stored in 'user_export_format').
+    
+    user_export_format: pulled from dropdown called by app.export_as()
+    data_to_export: xarray ds or da to export
```

Comment:
Probably worth updating the developers guide to note that final transform formats should be in xarray ds or da